openedx/core/lib/xblock_utils.py

```diff
@@ -211,6 +212,12 @@ def grade_histogram(module_id):
     return grades
 
 
+def sanitize_html_id(html_id):
+    """Template uses element_id in js function names, so can't allow dashes and colons"""
```

Comment:
Thanks for the explanation @vkaracic.

Here are our Python guidelines: https://github.com/edx/edx-platform/wiki/Python-Guidelines
The one-line comment here, and through the rest of the PR, should look as follows:
```
"""
Template uses element_id in js function names, so can't allow dashes and colons
"""
```

I imagine the html_id doesn't need to be reversible then, and hopefully it remains unique enough to serve its purpose.